Sources/iron/RenderPath.hx

```diff
@@ -687,17 +689,15 @@ class RenderPath {
 		if (height < 1) height = 1;
 		if (t.depth != null && t.depth > 1) { // 3D texture
 			// Image only
-			var img = Image.create3D(width, height, depth,
-				t.format != null ? getTextureFormat(t.format) : TextureFormat.RGBA32);
-			if (t.mipmaps) img.generateMipmaps(1000); // Allocate mipmaps
```

Comment:
Hm why is the `generateMipmaps()` call removed?

Reply:
Because we don't want to use mipmaps with 3D images.
It is unneeded and what we would gain in speed when sampling we loose in RAM.
I had done some tests with mipmaps and anyways it's not looking good, the original module as it's in WickedEngine doesn't use mipmaps.

Reply:
Why not just set `t.mipmaps` to `false` then? Right now you removed the ability to generate mipmaps of 3d images completely.

Reply:
I'm not certain but I don't think there was any mipmaps generating system in iron for 3D images before.
The mipmaps where generated in the renderpath.
But yes, if you prefer that it is done this way then OK.

Reply:
> I'm not certain but I don't think there was any mipmaps generating system in iron for 3D images before.

I checked some random old revision and looks like the `if (t.mipmaps) img.generateMipmaps(1000);` has been there for a long time:

https://github.com/armory3d/iron/blob/f263fa92b9a3d7759353479f167d8e5a7d6a4408/Sources/iron/RenderPath.hx#L658